pkgs/by-name/br/brioche/fetchers.nix

```diff
@@ -0,0 +1,20 @@
+# not a stable interface, do not reference outside the windmill package but make a copy if you need
+{
+  lib,
+  stdenv,
+  fetchurl,
+}:
+
+{
+  fetchLibrustyV8 =
```

Comment:
Can we dedup this? Already have a bunch of packages all vendoring their own implementation.

Reply:
True. I picked this from `windmill`.